deploy/crds/kieapp.crd.yaml

```diff
@@ -35,10 +35,6 @@ spec:
                 - rhpam-production-immutable
                 - rhpam-production
                 - rhpam-trial
-            kieDeployments:
-              type: integer
-              description: The number of distinct KIE server deployments
-              minimum: 1
```

Comment:
I'm thinking this should be optional if they provide a servers array

Reply:
It is optional. You should either pick the `server` for common config or `servers` for individual one. As you mentioned, a folder with examples is required at this point

Reply:
what if both are picked?